ApplicationPattern.yaml

```diff
@@ -8605,6 +8635,160 @@ paths:
         default:
           $ref: '#/components/responses/responseForErroredOamRequests'
 
+  /core-model-1-4:control-construct/logical-termination-point={uuid}/layer-protocol=0/tcp-server-interface-1-0:tcp-server-interface-pac/tcp-server-interface-configuration/description:
+    parameters:
+      - in: path
+        name: uuid
+        required: true
+        schema:
+          type: string
+          pattern: '^([a-z]{2,6})-([0-9]{1,2})-([0-9]{1,2})-([0-9]{1,2})-tcp-s-bm-000'
```

Comment:
The current pattern , will accept only "ro-2-0-0-tcp-s-bm-000". Since we have another tcp-server instance now , we require a pattern that accepts "ro-2-0-0-tcp-s-bm-001" too. Kindly please consider the following proposal, 
**Proposal :** 
Requesting to change the Pattern from , 
`pattern: '^([a-z]{2,6})-([0-9]{1,2})-([0-9]{1,2})-([0-9]{1,2})-tcp-s-bm-000'`
to 
`pattern: '^([a-z]{2,6})-([0-9]{1,2})-([0-9]{1,2})-([0-9]{1,2})-tcp-s-bm-00[0-1]`

Reply:
CTRL + H and changed as prosed. Thanks